### ISSUE TITLE
Page misalignment

### ISSUE THREAD

**Issue by kakaroto** [captn3m0/cosmere-books#5]
This is a weird new bug that seems to be a side effect of 46e302c4fb99b7c5957295ccff4ebf2fc5022907. The page is somehow misaligned in my reader. The text is properly aligned at the start of each chapter, then it slowly moves to the left with the next page shifting out from the right, the farther away you move from the chapter start.
It's hard to explain so I took some screenshots instead : 
![ob-issue-3](https://user-images.githubusercontent.com/27990/30927941-0c374382-a388-11e7-92e9-96e875078a43.png)
![ob-issue-3_2](https://user-images.githubusercontent.com/27990/30927940-0c36597c-a388-11e7-9515-635dbc212cc4.png)
![ob-issue-3_3](https://user-images.githubusercontent.com/27990/30927942-0c374a62-a388-11e7-9ab0-036593839a0d.png)

I don't understand why this would even happen in the epub reader since the reader is just supposed to render text. 
Hopefully, it makes sense to you.

**Comment by captn3m0** [comment]
It kinda does. I've set a global style on body for `margin`, which I think is getting re-applied on every page.

Can you try a rebuild after removing the following 3 lines from `style.css`:

1. https://github.com/captn3m0/cosmere-books/blob/46e302c4fb99b7c5957295ccff4ebf2fc5022907/style.css#L41
2. https://github.com/captn3m0/cosmere-books/blob/46e302c4fb99b7c5957295ccff4ebf2fc5022907/style.css#L113-L114

And see if it fixes the issue? I'll try installing Icecreame over wine to see if I can figure this out better.

**Comment by kakaroto** [comment]
Tried it, no change unfortunately.

**Comment by captn3m0** [comment]
Try removing the entire `body` section: https://github.com/captn3m0/cosmere-books/blob/46e302c4fb99b7c5957295ccff4ebf2fc5022907/style.css#L36-L42

If that doesn't work, it has to be the `text-align: justify`, which will get us back to the center aligned text. Will see if I can find an epub css template.

**Comment by kakaroto** [comment]
Nope, I removed the body and the text-align and it broke the justification but the page overlap remained. I then removed the stylesheet argument and it fixed it (but justify broke of course).
https://github.com/captn3m0/cosmere-books/blob/46e302c4fb99b7c5957295ccff4ebf2fc5022907/methods.rb#L44

Then I started to progressively delete anything I saw in the css (I don't understand any of it) until it worked.. I had to do a lot of tests before I realized that if I don't delete the ebook from the reader, then empty the trash, then close the app before opening the file, then it would somehow load a cached version of it... 
Anyways, I found the faulty line : https://github.com/captn3m0/cosmere-books/blob/46e302c4fb99b7c5957295ccff4ebf2fc5022907/style.css#L108
Just delete that line (which ironically says "increasingly indent" ) and it will fix it!
